Tests/SwiftDiagnosticsTest/DiagnosticsFormatterTests.swift

```diff
@@ -0,0 +1,81 @@
+//===------------------ DiagnosticsFormatterTests.swift -------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import XCTest
+@_spi(Testing) import SwiftDiagnostics
```

Comment:
I don’t think this needs to be `@_spi(Testing)` because we don’t have any types or functions in SwiftDiagnostics annotated with `@_spi(Testing)`